tests/unit/moduleapi/list.tcl

```diff
@@ -70,49 +80,37 @@ start_server {tags {"modules"}} {
         # delete from start of list (index 0)
         r del l
         r rpush l x y z
-        set reply [r list.edit l dd]
-        assert_equal [lindex $reply 3] 0
-        assert_equal [lindex $reply 5] {z}
+        verify_list_edit_reply [r list.edit l dd] 0 2 0 0 z
```

Comment:
Cool!

But now again it is hard to remember which digit is what without looking at the definition of verify_list_edit_reply. Maybe something like ...

    verify_list_edit_reply [r list.edit l dd] {i 0 d 2 r 0 index 0 entry z}

... is more directly readable?

Perhaps I should be happy already. The bug is fixed and this review is going too far. :-)

Reply:
Done.